### ISSUE TITLE
Milestone 2 Feedback

### ISSUE THREAD

**Issue by KMAyman** [data301-2021-summer2/project-group26-project#1]
Hello, 

My name is Ayman and I will be giving you some feedback on your project. 

Overall, the project is a fantastic start and there are some suggestions I have that will greatly improve the quality of your project.

I have divided up the feedback into several sections so it is easier for you to implement.

## Task 1

Good work.

## Task 2

Good work.

## Task 3

Provide some description about why you did that particular graph. There is also a compilation error in your milestone2.ipynb file.

## Task 4

I couldn't find the research question. Although I can see the answer in your task4 file but no question. Try to describe your question first and then try to answer it with appropriate plots. It's hard to follow what graph you are talking about just by reading it.

**Comment by bmatmti** [comment]
Thank you for your feedback. I have implemented your suggestions
